devices/src/virtio/mmio.rs

```diff
@@ -1,4 +1,3 @@
-// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is this removed?

Reply:
Because of force pushing in hurry. Sorry.
Please re-review.